backend/LexBoxApi/Controllers/ProjectController.cs

```diff
@@ -232,6 +232,17 @@ private async Task StreamHttpResponse(HttpContent hgResult)
         await hgResult.CopyToAsync(writer.AsStream());
     }
 
+    [HttpGet("getLdmlZip")] // TODO: Discuss endpoint name, and whether it should be GET or POST, at next opportunity
+    [AdminRequired] // TODO: Decide on permissions, because we don't want everyone triggering this
```

Comment:
admin only is a good choice, it won't be exposed in the UI so I'm not worried about our admins executing it

Reply:
TODO comment removed in commit 72f1a52e.